src/state-summary/state-card-display.js

```diff
@@ -63,6 +75,11 @@ class StateCardDisplay extends LocalizeMixin(PolymerElement) {
         type: Boolean,
         value: false,
       },
+      rtl: {
+        type: Boolean,
+        reflectToAttribute: true,
+        computed: "computeRTL(hass)",
```

Comment:
All method used privately by this class should be prefixed, so name it `_computeRTL`